ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -418,6 +422,15 @@ public void setLineHeight(int lineHeight) {
     markUpdated();
   }
 
+  @ReactProp(name = ViewProps.ALLOW_FONT_SCALING)
+  public void setAllowFontScaling(@Nullable Boolean allowFontScaling) {
+    mAllowFontScaling = allowFontScaling != null && allowFontScaling.booleanValue();
```

Comment:
Good call.  1 is resolved per your recommendation.  I also made sure 2 and 3 are working with my latest commit.  Here is a screen recording of it working with both dynamic font size as well as toggling allowFontScaling between true and false

https://youtu.be/AjDurnKfYKY